cairo-contracts/packages/clients/src/cometbft/client_state.cairo

```diff
@@ -39,4 +39,17 @@ pub impl CometClientStateImpl of CometClientStateTrait {
     fn freeze(ref self: CometClientState, freezing_height: Height) {
         self.status = Status::Frozen(freezing_height);
     }
+
+    fn substitute_client_matches(
+        ref self: CometClientState, other_client_state: CometClientState,
+    ) -> bool {
+        let mut substitute_client_state = other_client_state.clone();
```

Comment:
isn't this unnecessary cloning? unless you pass `other_client_state: @CometClientState`.

Reply:
Fixed in bf0f1f963677e36c28a4f67cd39291e8375e9977